.github/dependabot.yml

```diff
@@ -16,10 +16,38 @@ updates:
       - "release-note-none"
     open-pull-requests-limit: 5
 
-  - package-ecosystem: "npm"
-    directory: "/site"
+  - package-ecosystem: "gomod"
+    directory: "/"
+    schedule:
+      interval: daily
+    labels:
+      - "release-note-none"
+    open-pull-requests-limit: 5
+    target-branch: "release-1.23"
```

Comment:
is it possible to limit the dependabot only upgrading k8s dependencies for matched versions? (e.g. release-1.23 should only use k8s.io/cloud-provider v0.23.x)

Reply:
For now, dependabot doesn't support it. We can ignore k8s related dependencies and manually update them.

Reply:
yes, looks good